srcpkgs/base-files/files/_vkpurge

```diff
@@ -0,0 +1,29 @@
+#compdef vkpurge
+
+_vkpurge() {
+	local line state
+
+	_arguments -C \
+		'1: :->subcmd' \
```

Comment:
> A function calling `_arguments` with at least one action containing a ‘->string' must therefore declare appropriate local parameters:
> 
>     local context state state_descr line
>     typeset -A opt_args
> 
> to prevent `_arguments` from altering the global environment.